test/SlowTests/Issues/RavenDB_16950.cs

```diff
@@ -0,0 +1,51 @@
+﻿using System;
+using System.Collections.Generic;
+using Xunit;
+using FastTests;
+using Tests.Infrastructure;
+using Xunit.Abstractions;
+
+namespace SlowTests.Issues
+{
+    public class RavenDB_16950 : RavenTestBase
+    {
+        public RavenDB_16950(ITestOutputHelper output) : base(output)
+        {
+        }
+
+        [Fact]
+        public void CheckIfDecimalValueNotChangesAfterComputation()
+        {
+            int changesCounter;
+            using (var store = GetDocumentStore())
+            {
+                store.Maintenance.Send(new CreateSampleDataOperation());
```

Comment:
Let's avoid doing that if possible. Takes a while and you need only 1 entry in the test.